pages/tutorial.js

```diff
@@ -77,11 +80,12 @@ const tutorialView = (state, prev, send) => {
     },
 
     panels: [
-      blocklyHtml,
       [
-        gameHtml,
-        goalProgressHtml
-      ]
+        instructionHtml,
+        goalHtml
+      ],
+      blocklyHtml,
+      gameHtml
     ]
   })
```

Comment:
Could you add a button to quickly hide/show the new panel with the instructions and the goal? (kind of like a slide-in/hamburger menu). As of state now the view feels really squished into the browser with 3 panels active.  
Yes I know they are resizable and the player can simply resize it out, but i really think the instruction part should be a slide-in. (Maybe upon hovering over the slide-in button we could display the goals again without opening the slide-in)  

![image](https://cloud.githubusercontent.com/assets/5808849/23092905/44625eec-f5d5-11e6-92a2-9b6269d0a6a0.png)